.github/workflows/tests.yml

```diff
@@ -35,6 +38,7 @@ jobs:
             libdbus-1-dev \
             libdqlite-dev \
             liblxc-dev \
+            lxc-templates \
```

Comment:
Why do we need that?

Reply:
Because of the unit test:

https://github.com/lxc/lxd/blob/master/lxc-to-lxd/main_migrate_test.go#L214